### PR TITLE
[MRG] Add doc strings for text models.

### PR DESCRIPTION
**Pull request by satyakesav**
The details of the pull request:
Add doc strings for pre-trained text modules and text classifier module.

### CODE REVIEW

File: autokeras/nn/model_trainer.py [hunk 1]
```diff
@@ -356,7 +373,22 @@ def _train(self, epoch):
 
 
 class BERTTrainer(ModelTrainerBase):
+    """A ModelTrainer for the Google AI's BERT model. Currently supports only classification task.
```
Comment (jhfjhfj1) [inline autokeras/nn/model_trainer.py]: Add an empty line here.

File: autokeras/nn/model_trainer.py [hunk 2]
```diff
@@ -356,7 +373,22 @@ def _train(self, epoch):
...
+    Attributes:
+        model: Type of BERT model to be used for the task. E.g:- Uncased, Cased, etc.
+        output_model_file: File location to save the trained model.
+        num_labels: Number of output labels for the classification task.
+    """
+
     def __init__(self, train_data, model, output_model_file, num_labels, loss_function=None):
+        """Initialize the BERTTrainer.
```
Comment (jhfjhfj1) [inline autokeras/nn/model_trainer.py]: Add an empty line here.

File: autokeras/nn/model_trainer.py [hunk 3]
```diff
@@ -383,6 +415,15 @@ def train_model(self,
                     max_iter_num=None,
                     max_no_improvement_num=None,
                     timeout=None):
+        """Train the model.
+        Train the model with max_iter_num.
+        Args:
+            timeout: timeout in seconds
+            max_iter_num: An integer. The maximum number of epochs to train the model.
+            max_no_improvement_num: An integer. The maximum number of epochs when the loss value doesn't decrease.
```
Comment (jhfjhfj1) [inline autokeras/nn/model_trainer.py]: Add an empty line here.

**Comment by satyakesav** [comment]
> Hi Satya, please follow my comments to add all the empty lines. I didn't comment all the part that need an empty line, but you see my point, just use empty lines to separate the sections. Only in this way, the autogen.py can collect those docs correctly to generate the website.
> 
> Thanks

I think I understood what you said. I have made the changes. Please let me know if you think anything else needs to be changed.

**Pull request merged**